source/help/registerhelp.md

```diff
@@ -1,48 +1,46 @@
 # Author Registration
 
 To begin registration, visit the [registration form](https://arxiv.org/user/register).
-First, we will ask for your e-mail and you will get to pick a user name
-and password. Your user name must be unique and cannot be changed.  
-Recommendations for arXiv password:
 
--  8 characters or more in length
+Your username must be unique and cannot be changed.  
 
-- May contain upper/lowercase letters, numbers, and one or more underscores (_) - one underscore is required
+arXiv registration also asks for your name and your Organization. If you are unaffiliated you may enter "Indepdendent" as your organization.
+
+Select which archives you are most likely to submit to. This will filter the category options available if you submit an article in the future. 
+
+We will then verify the email for your account.
+
+Please [review account related policies](policies/identity_and_affiliation.md).
 
-- May not contain any other special characters besides underscore - no other punctuation or symbols are allowed (ex. $, ., -,@)
 
-Some successful examples could be: “Pass_Word_123” or “pAssWoRd_123”
+## Account Password
 
-Second, we will ask you for some personal information such as your name,
-your institution (or organization) and which archives you are most
-likely to submit to. We will then verify the email for your account.
+Account password requirements:
 
-*Policy Notes:*
+-  8 characters or more in length
+
+- May contain upper/lowercase letters, numbers, and one or more underscores (_) - one underscore is required
```

Comment:
Suggestion: Separate this into two parts and spell out "uppercase". 

- May contain uppercase and lowercase letters, and numbers
- Must contain at least one underscore (_)